packages/plugin-dev-phone/src/commands/dev-phone.ts

```diff
@@ -266,7 +266,7 @@ class DevPhoneServer extends TwilioClientCommand {
         try {
             const functionServices = await this.twilioClient.serverless.services.list()
             const devPhoneFunctionServices = functionServices.filter((functionServices: ServerlessServiceInstance) => {
-            return functionServices.friendlyName !== null && functionServices.friendlyName.startsWith('dev-phone')
+            return functionServices.friendlyName !== null && functionServices.friendlyName.startsWith(this.devPhoneName)
```

Comment:
Looks good and works correctly, but could you update the associated text on lin 273 to be more accurate to what's happening? I'll include the line for each, it's a bit tedious:

```
console.log(`🚮 Removing Serverless Functions for ${this.devPhoneName}`);
```